exec/java-exec/src/test/java/org/apache/drill/test/ClusterTest.java

```diff
@@ -125,4 +139,29 @@ public static void run(String query, Object... args) throws Exception {
   public QueryBuilder queryBuilder( ) {
     return client.queryBuilder();
   }
+
+  /**
+   * Generates a compressed file for testing
+   * @param fileName the input file to be compressed
+   * @param codecName the CODEC to be used for compression
+   * @param outFileName the output file name
+   * @throws IOException Throws IO exception if the file cannot be found or any other IO error
+   */
+  public void generateCompressedFile(String fileName, String codecName, String outFileName) throws IOException {
```

Comment:
Please make it static and move to a more suitable place, for example to `QueryTestUtil`.

Reply:
Moved to `QueryTestUtil`

Reply:
Note:  By moving this class, this PR also addresses Drill-7600: https://issues.apache.org/jira/browse/DRILL-7600.